.github/workflows/translations-pull.yml

```diff
@@ -3,11 +3,10 @@
 name: 'Pull translations'
 
 on:
+  schedule:
+    - cron: '0 */2 * * *' # Every two hours.
```

Comment:
I really think this should be daily, otherwise we're potentially gonna litter ourselves with PR's when we have some inflight work.

Reply:
The action force pushes the existing PR so it won't ever create more than one